src/components/EnvironmentOverride/DeploymentTemplateOverrideForm.tsx

```diff
@@ -245,7 +245,7 @@ export default function DeploymentTemplateOverrideForm({
             } else {
                 dispatch({
                     type: DeploymentConfigStateActionTypes.tempFormData,
-                    payload: YAML.stringify(deploymentTemplateResp.result.envOverrideValues),
+                    payload: YAML.stringify(deploymentTemplateResp.result?.envOverrideValues),
```

Comment:
stringify will return undefined if the input is undefined which can be here if result is undefined. Later this value will be parsed using JSON.parse which will through error if undefined is passed to it. Best course of action would be to not call dispatch if result is undefined.